org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/StringFragment.java

```diff
@@ -0,0 +1,181 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.jdt.core.dom;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * String fragment node.
+ *
+ * @since 3.37
+ * @noinstantiate This class is not intended to be instantiated by clients.
+ */
+@SuppressWarnings("rawtypes")
+public class StringFragment extends Expression {
+
+	/**
+	 * The "escapedValue" structural property of this node type (type: {@link String}).
+	 * @since 3.0
```

Comment:
since the class itself is from 3.37 and its new, this should be removed - to be checked with API analyzer